### ISSUE TITLE
cocoapods task generateDefs generates invalid defs for React

### ISSUE THREAD

**Issue by kdawgwilk** [JetBrains/kotlin-native#2872]
I created a sample repo that reproduces the issue here https://github.com/kdawgwilk/KotlinNativeBug

```

> Task :generateDefReact
> Task :cinteropReactIos
e: /path/to/project/KotlinNativeBug/SharedLib/build/classes/kotlin/ios/main/SharedLib-cinterop-React.klib-build/kotlin/cocoapods/React/React.kt: (2970, 23): 'toString' hides member of supertype 'NSObject' and needs 'override' modifier
e: /path/to/project/KotlinNativeBug/SharedLib/build/classes/kotlin/ios/main/SharedLib-cinterop-React.klib-build/kotlin/cocoapods/React/React.kt: (14752, 25): 'public final var attributedText: NSAttributedString defined in cocoapods.React.RCTUITextView' clashes with 'public final var attributedText: NSAttributedString? defined in cocoapods.React.RCTBackedTextInputViewProtocolProtocol': property types do not match

```

**Comment by SvyatoslavScherbina** [comment]
Confirmed, thanks.

**Comment by SvyatoslavScherbina** [comment]
Btw, I was able to get only the second error.
Could you show me the class from `/path/to/project/KotlinNativeBug/SharedLib/build/classes/kotlin/ios/main/SharedLib-cinterop-React.klib-build/kotlin/cocoapods/React/React.kt` containing line 2970?

**Comment by kdawgwilk** [comment]
I think the first error I was seeing was in an older version of React 0.57.8 so it can be ignored

**Comment by SvyatoslavScherbina** [comment]
Will be fixed in 1.3.31.
